lib/notifier.js

```diff
@@ -298,11 +300,18 @@ function addItem(item, callback) {
       if (err) {
         return callback(err);
       }
-      apiClient.postItem(data, function (err, resp) {
-        if (typeof callback === 'function') {
-          callback(err, data, resp);
+      if(SETTINGS.enabled){
```

Comment:
I would move this check to happen before the call to `buildItemData()` since we are basically neutering the call to `addItem()` if `options.enabled === false`.